protocols/sftp/index.md

```diff
@@ -391,6 +391,19 @@ Cyberduck refuses to connect if there are malformed entries in your `known_hosts
 
 Symbolic links only work within Mountain Duck if the target of the symbolic link is available within the mounted path.
 
+````{tabs}
+```{group-tab} macOS
+
+Symlinks can only be displayed as such if the target is included in the mount, otherwise they are displayed and function as normal folders.
```

Comment:
```suggestion
Symlinks can only be displayed as such if the target is included in the mount, otherwise they are displayed and function as a regular file or folder.
```